src/search/createGeoJSONBloodhound.js

```diff
@@ -4,7 +4,7 @@
 import olFormatGeoJSON from 'ol/format/GeoJSON.js';
 import * as olObj from 'ol/obj.js';
 
-import 'corejs-typeahead';
+import Bloodhound from 'corejs-typeahead';
```

Comment:
same

Reply:
Why this?

Reply:
Hmm, I don't remember. @llienher, let's try to remove these changes and see what happens.